### PR TITLE
Create DebugWindowsNode.ps1

### PR DESCRIPTION
**Pull request by jayunit100**

### CODE REVIEW

File: hack/DebugWindowsNode.ps1 [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+$currentVersion = Get-Item 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion'
+$OSProductName = $currentVersion.GetValue('ProductName')
+$OSBuildLabel = $currentVersion.GetValue('BuildLabEx')
+Write-Output "Container Host OS Product Name: $OSProductName"
+Write-Output "Container Host OS Build Label: $OSBuildLabel"
+
+Describe "Windows Version and Prerequisites" {
+    $buildNumber = (Get-CimInstance -Namespace root\cimv2 Win32_OperatingSystem).BuildNumber
+    It "Is Windows Server 2019" {
+        $buildNumber -eq 17763 | Should Be $true
```
Comment (marosset) [inline hack/DebugWindowsNode.ps1:14]: Can you make this check -ge 17763?
We support k8s all Windows server versions including/after Windows Server 2019.

File: hack/DebugWindowsNode.ps1 [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        # This also captures 'docker info' and 'docker version' output to be shown later
+        {
+            $versionText = & docker.exe version --format '{{ json . }}'
+            $global:DockerVersion = $versionText | ConvertFrom-Json
+        } | Should Not Throw
+    }
+
+    It "Should be a supported version" {
+        $supportedVersions = @(
+            "18.09.1",
```
Comment (marosset) [inline hack/DebugWindowsNode.ps1:61]: Maybe drop the 18.* versions?
I think we want everyone on 19.03.*

**Comment by jayunit100** [comment]
ok comments addressed thx mark

File: hack/DebugWindowsNode.ps1 [hunk 3]
```diff
@@ -0,0 +1,79 @@
+Write-Output "Checking for common problems with Windows Kubernetes nodes"
+Î
```
Comment (aravindhp) [inline hack/DebugWindowsNode.ps1:2]: I think this is a stray character and needs to be removed.

Comment (jayunit100) [reply hack/DebugWindowsNode.ps1:2]: ahhhh windows copy paste error :)

**Comment by jsturtevant** [comment]
Did we want to add a check for Containerd as well?  Or have a separate script?

**Comment by jayunit100** [comment]
im happy to add an either or for containerd.maybe in a follow on pr?

**Pull request merged**